src/applications/letters/actions/letters.js

```diff
@@ -74,7 +74,10 @@ export function getLetterList(dispatch) {
       } else {
         dispatch({ type: GET_LETTERS_FAILURE });
       }
-      throw new Error(`vets_letters_error_getLetterList: ${status}`);
+      const lettersError = new Error('vets_letters_error_getLetterList');
+      Raven.captureException(lettersError, {
+        fingerprint: ['{{ default }}', status],
```

Comment:
@cvalarida I _think_ this is what we want to do, if I understand the [docs](https://docs.sentry.io/data-management/rollups/?platform=javascript#grouping-priorities) correctly. I added it to all the letters calls using it, since it seems we were running into the same problem, albeit not as severely, in the other error logging

Reply:
Ahh, yes. This looks like exactly what we want to do. To keep the same behavior as we had before, we probably still want to throw an error. Unless you've determined it doesn't cause any regressions to not throw it, of course.